src/Relation/Binary/Properties/DecSetoid.agda

```diff
@@ -36,8 +37,19 @@ open SetoidProperties setoid
   ; cotrans = ≉-cotrans
   }
 
+≉-tight : Tight _≈_ _≉_
+≉-tight x y = decidable-stable (x ≟ y)
```

Comment:
Missing addition from Changelog.

Reply:
yeah, I wasn't sure how to record this... it's a *redefinition*, rather than an addition, because the definition of `Tight` has been changed in `Relation.Binary.Definitions` to drop the redundant second conjunct, which is equivalent to `Irreflexive`, as noted on the issue.

What's the best way to do this?